src/didactWebView.ts

```diff
@@ -363,8 +372,14 @@ export class DidactWebviewPanel {
 			stylesheetHtml + 
 			`<script defer="true" src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
 			</head>
-		<body class="content">` + didactHtml + 
-		`<script nonce="${nonce}" src="${scriptUri}"/>
+		<body class="content">
+			<div class="didactHeader">
+			<img class="didactHeaderImage" src="https://raw.githubusercontent.com/redhat-developer/vscode-didact/master/icon/logo.png"/></div>
```

Comment:
> It does make the 'Welcome to Didact' banner look huge tho (does your banner look big in didact?)
> I wonder about the title, separate issue methinks ;)

The banner is a moot point. Just wanted a simple graphic at the top. It's not that huge. :+1: 

The title comes automatically from the first H1/H2 title found in the document. It will vary from tutorial to tutorial and I thought that was the safest way to name the tab. It gets really long sometimes, but it works.

And if you have suggestions regarding that demo we should definitely put 'em in a separate issue. :)

Reply:
the last comment is in the wrong comment thread

Reply:
keeping the header?

Reply:
Rebase/merge glitch